collective/dancing/composer.py

```diff
@@ -574,3 +576,21 @@ def at_exit():
         "StubSMTPMailer shutting down, sent %s messages to %s recipients.\n" %
         (StubSMTPMailer.sent, len(StubSMTPMailer.recipients)))
     StubSMTPMailer.logfile.close()
+
+
+class Dispatch(collective.singing.mail.Dispatch):
+    """An IDispatcher registered for ``email.message.Message`` that'll
+    send e-mails using the default Plone MailHost. """
+    interface.implements(collective.singing.interfaces.IDispatch)
+    component.adapts(Message)
+
+    def __call__(self):
+        msg = self.message
+        delivery = component.getUtility(Products.CMFPlone.interfaces.IPloneSiteRoot).MailHost
```

Comment:
Please wrap your lines to fit into 80-char (see [Plone coding conventions](http://ploneapi.readthedocs.org/en/latest/contribute/conventions.html#breaking-lines))